api_tests/dry/rfc003_swap_with_ip.ts

```diff
@@ -0,0 +1,101 @@
+import { Actor } from "../lib/actor";
+import { HarnessGlobal, sleep } from "../lib/util";
+import { request } from "chai";
+import "chai/register-should";
+import { toWei } from "web3-utils";
+import "../lib/setupChai";
+
+declare var global: HarnessGlobal;
+
+(async function() {
+    const alpha_ledger_name = "bitcoin";
+    const alpha_ledger_network = "regtest";
+
+    const beta_ledger_name = "ethereum";
+    const beta_ledger_network = "regtest";
+
+    const alpha_asset_name = "bitcoin";
+    const alpha_asset_reasonable_quantity = "100000000";
+
+    const beta_asset_name = "ether";
+    const beta_asset_quantity = toWei("10", "ether");
+
+    const alpha_expiry = new Date("2080-06-11T23:00:00Z").getTime() / 1000;
+    const beta_expiry = new Date("2080-06-11T13:00:00Z").getTime() / 1000;
+
+    const alice = new Actor("alice", global.config, global.project_root);
+    const bob = new Actor("bob", global.config, global.project_root);
+    const alice_final_address = "0x00a329c0648769a73afac7f9381e08fb43dbea72";
+    const bob_peer_id = await bob.peerId();
+    const peer_input = {
+        peer_id: await alice.peerId(), // Incorrect peer id on purpose to see if Bob still appears in GET /swaps
```

Comment:
:mega: This is the crucial line of the whole test, yet that is not obvious at all. I think we shouldn't make this one dynamic. Just take a random one and put it in as a string.

If we inline all the other initializing logic, than it should be more obvious, what we are tring to do here.

Reply:
any suggestion on how to easily make a random peer id?

Reply:
> The best way to test this would be to disable any discovery service (mDNS).

Seems to be reasonable. I'll create a followup ticket that includes fixing this test.

Reply:
Done: https://github.com/comit-network/comit-rs/issues/1007